<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindReplacePane</class>
 <widget class="QWidget" name="FindReplacePane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolBar" name="toolBar">
     <property name="movable">
      <bool>false</bool>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <addaction name="actionClear"/>
     <addaction name="actionCollapse_All"/>
     <addaction name="actionExpand_All"/>
     <addaction name="separator"/>
     <addaction name="actionStop"/>
     <addaction name="actionSearch"/>
     <addaction name="separator"/>
     <addaction name="actionPreserve_Results"/>
    </widget>
   </item>
   <item>
    <widget class="wProgressBar" name="widget" native="true"/>
   </item>
   <item>
    <widget class="QTreeView" name="treeView">
     <property name="headerHidden">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string/>
     </property>
     <property name="margin">
      <number>4</number>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="../../r.qrc">
     <normaloff>:/Resource/icons/Stop_16x.svg</normaloff>:/Resource/icons/Stop_16x.svg</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="icon">
    <iconset resource="../../r.qrc">
     <normaloff>:/Resource/icons/Refresh_16x.svg</normaloff>:/Resource/icons/Refresh_16x.svg</iconset>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </action>
  <action name="actionPreserve_Results">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../r.qrc">
     <normaloff>:/Resource/icons/Lock_16x.svg</normaloff>:/Resource/icons/Lock_16x.svg</iconset>
   </property>
   <property name="text">
    <string>Preserve Results</string>
   </property>
   <property name="toolTip">
    <string>Preserve Results</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="../../r.qrc">
     <normaloff>:/Resource/icons/CleanData_16x.svg</normaloff>:/Resource/icons/CleanData_16x.svg</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionExpand_All">
   <property name="icon">
    <iconset resource="../../r.qrc">
     <normaloff>:/Resource/icons/ExpandAll_16x.svg</normaloff>:/Resource/icons/ExpandAll_16x.svg</iconset>
   </property>
   <property name="text">
    <string>Expand All</string>
   </property>
  </action>
  <action name="actionCollapse_All">
   <property name="icon">
    <iconset resource="../../r.qrc">
     <normaloff>:/Resource/icons/CollapseAll_16x.svg</normaloff>:/Resource/icons/CollapseAll_16x.svg</iconset>
   </property>
   <property name="text">
    <string>Collapse All</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>wProgressBar</class>
   <extends>QWidget</extends>
   <header location="global">w_progress_bar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../r.qrc"/>
 </resources>
 <connections/>
</ui>
